calculatorApi1.js

```diff
@@ -0,0 +1,50 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+app.use(cors())
+const port = 3000
+
+const add = (arg1, arg2) =>{
```

Comment:
Typically you would want to turn your values into numbers before inputting them into a function that adds